packages/components/order-tracking/src/order-tracking.html

```diff
@@ -0,0 +1,106 @@
+<div class="order-tracking">
+
+    <oui-back-button data-on-click="$ctrl.onBackButtonClick()">
+        <span data-translate="order_tracking_title"
+              data-translate-values="{ orderId: $ctrl.orderId }"></span>
+    </oui-back-button>
+
+    <div class="text-center">
+        <oui-spinner data-ng-if="$ctrl.isLoading" size="l">
+        </oui-spinner>
+    </div>
+
+    <oui-message type="error" data-ng-if="$ctrl.error">
+        <p>
+            <strong data-translate="order_tracking_error"></strong>
+        </p>
+        <p>
+            <span data-ng-bind="$ctrl.error.message"></span>
+            <br />
+            <small data-ng-bind="$ctrl.error.queryId"></small>
+        </p>
+        <oui-button data-variant="secondary"
+                    data-text="{{ 'order_tracking_try_again' | translate }}"
+                    data-on-click="$ctrl.refresh()">
+        </oui-button>
+    </oui-message>
+
+    <div data-ng-if="$ctrl.order">
+
+        <oui-spinner data-ng-if="!$ctrl.orderFollowUp" size="m">
+        </oui-spinner>
+        <div class="oui-progress-tracker"
+             data-ng-if="$ctrl.orderFollowUp">
+            <ol class="oui-progress-tracker__steps">
+                <li class="oui-progress-tracker__step"
+                    data-ng-repeat="followUp in $ctrl.orderFollowUp"
+                    data-ng-class="{
+                    'oui-progress-tracker__step_complete': followUp.status === 'DONE',
+                    'oui-progress-tracker__step_active': followUp.status === 'DOING',
+                    'oui-progress-tracker__step_error': followUp.status === 'ERROR',
+                    }">
+                    <span class="oui-progress-tracker__status">
+                        <span class="oui-progress-tracker__label"
+                              data-translate="{{:: 'order_tracking_step_' + followUp.step }}">
+                        </span>
+                    </span>
+                </li>
+            </ol>
+        </div>
+
+        <div class="row">
+            <div class="col-md-6 mt-4">
+                <oui-tile data-heading="{{:: 'order_tracking_status' | translate }}">
+                    <oui-spinner data-ng-if="!$ctrl.orderHistory" size="m">
+                    </oui-spinner>
+                    <oui-tile-definition data-ng-repeat="event in $ctrl.orderHistory.slice(0, $ctrl.maxHistoryLength)">
+                        <oui-tile-description>
+                            <div class="row">
+                                <strong class="text-nowrap col-lg-4 col-xs-12"
+                                        data-ng-class="{ 'oui-color-regent': $index > 0 }">
+                                    {{ event.date | date:'medium' }}
```

Comment:
```suggestion
                                    <span data-ng-bind="event.date | date:'medium'></span>
```
is better for performances :)